core/tests/webdriverio_utils/AdminPage.js

```diff
@@ -185,6 +251,31 @@ var AdminPage = function() {
     await this.saveChangeOfFeature(featureElement);
   };
 
+  this.enableFeatureForDev = async function(featureElement) {
+    await this.removeAllRulesOfFeature(featureElement);
+
+    await action.click(
+      'Add feature rule button',
+      featureElement
+        .$(addFeatureRuleButtonLocator)
+    );
+
+    await waitFor.visibilityOf(
+      featureElement.$(valueSelectorLocator),
+      'Value Selector takes too long to appear'
+    );
+    await (featureElement.$(valueSelectorLocator)).selectByVisibleText(
+      'Enabled');
+
+    await action.click(
+      'Add condition button',
+      featureElement
+        .$(addConditionButtonLocator)
+    );
+
```

Comment:
[This](https://github.com/oppia/oppia/blob/develop/core/tests/protractor_utils/AdminPage.js#L275) seems to be missing.

Reply:
The link you are pointing to is from `enableFeatureForProd` function but here we have `enableFeatureForDev` function. And in this function we do not have sendKeys beacause by default the value of dropdown is always 'dev'.